### ISSUE TITLE
Create a flatpak package for BorgBackup+GUI

### ISSUE THREAD

**Issue by thecursedfly** [MTrage/Borg-BackUP-GUI#9]
Hi,

It would be great if you could package the GUI together with BorgBackup in a flatpak package (see https://flatpak.org/ for info/documentation).
That way me and other non techy users would be able to use this data backup solution on their computer without having to build from source or install separate components.
I recommend a flatpak package because it is distribution independent, avoids dependency problems and incompatibilities with older/newer Linux distribution versions, and allows sandboxing for additional security.

Thanks

**Comment by MTrage** [comment]
Yes this is a very good idea, I already had this thought, because I am a **flatpak** fan. The inclusion of **Borg Backup** and **Borg BackUP GUI** in a unified package will (to my knowledge) be possible as both use different license models. But the actual installation of **Borg Backup** is very easy and also available for countless distros. 

I will probably create a package in a few weeks if there are still some things on my todo list regarding **BORG BackUP GUI** done.
